examples/vision/shiftvit.py

```diff
@@ -543,6 +566,24 @@ def call(self, x, training=False):
             x = self.patch_merge(x)
         return x
 
+    # Since this is a custom layer, we need to overwrite get_config()
+    # so that model can be easily saved & loaded after training
+    def get_config(self):
```

Comment:
Fix for this error while saving model:

_"NotImplementedError: 
Layer StackedShiftBlocks has arguments ['self', 'epsilon', 'mlp_dropout_rate', 'num_shift_blocks', 'stochastic_depth_rate', 'is_merge', 'num_div', 'shift_pixel', 'mlp_expand_ratio']
in `__init__` and therefore must override `get_config()`"_